api/applications/applicationRouter.js

```diff
@@ -208,29 +205,33 @@ router.get('/profileId/:id', checkApplicationExists, checkRole, (req, res) => {
 
 // create a new user profile and application ticket
 //this only works for the mentor application because we are passing the mentorApplicationSchema directly (6/4/2022)
-router.post(
-  '/new/:role',
-  validation(),
-  createProfile,
-  cacheSignUpData,
-  sendData,
-  (req, res, next) => {
-    //this applicationTicket object works for the existing backend db schema for "tickets". Both likely need to be updated (6/4/2022)
-    const applicationTicket = {
-      ticket_status: 'pending',
-      //unsure of correct value for 'ticket_type' to indicate this is an application (6/4/2022)
-      ticket_type: 1,
-      ticket_subject: 'application',
-      requested_for: req.body.profile_id,
-      submitted_by: req.body.profile_id,
-    };
-    Application.add(applicationTicket)
-      .then(() => {
-        res.status(201).json({ message: 'Application has been submitted' });
-      })
-      .catch(next);
+router.post('/new/:role', validation(), async (req, res, next) => {
```

Comment:
This endpoint was refactored in BL-1040 (not yet merged) - changes to the frontend do not work without this.